src/lib/sorting/tag-sort.ts

```
@@ -0,0 +1,352 @@
+  // Parsing JSON sécurisé avec gestion d'erreurs spécifique
+  let postsRaw: unknown;
+  let translationsRaw: unknown;
+
+  try {
+    postsRaw = JSON.parse(postsTextContent);
+  } catch (error) {
+    console.error("Tag sort: Failed to parse posts JSON data", {
+      error: error instanceof Error ? error.message : String(error),
+      textContent: postsTextContent.substring(0, 100) + "..." // Log excerpt for debugging
+    });
+    return;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)